<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tagEditWidget</class>
 <widget class="QWidget" name="tagEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>835</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>tagEditor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../image.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(200, 200, 200);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QAxWidget" name="axWidget">
         <property name="control" stdset="0">
          <string>{6bf52a52-394a-11d3-b153-00c04f79faa6}</string>
         </property>
         <property name="styleSheet" stdset="0">
          <string notr="true">	background-color: rgb(60,60,60);
	color: #fff;
	font: 9pt &quot;ç­‰çº¿l&quot;;
	border:0px solid rgb(0,0,0);
	border-radius:5px;
</string>
         </property>
         <property name="sizePolicy" stdset="0">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>750</width>
           <height>140</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>140</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255,0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="3">
           <widget class="QPushButton" name="targetButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget
{
	background-color:rgb(40,40,40,0);
	font: 9pt &quot;Arial&quot;;
}
QWidget:hover
{
	font-size:12pt;
}
</string>
            </property>
            <property name="text">
             <string>ðŸ“‚</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0,0,0);
font: 87 9pt &quot;Arial Black&quot;;</string>
            </property>
            <property name="text">
             <string>source</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="sourceButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget
{
	background-color:rgb(40,40,40,0);
	font: 9pt &quot;Arial&quot;;
}
QWidget:hover
{
	font-size:12pt;
}
</string>
            </property>
            <property name="text">
             <string>ðŸ“‚</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0,0,0);
font: 87 9pt &quot;Arial Black&quot;;</string>
            </property>
            <property name="text">
             <string>target</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0,0,0);
font: 87 9pt &quot;Arial Black&quot;;</string>
            </property>
            <property name="text">
             <string>Collection</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget
{
	color: #fff;
	font: 9pt &quot;ç­‰çº¿l&quot;;
	border:2px solid rgb(0,0,0);
}	

QCheckBox::indicator
{
	width:30px;
	height:30px;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="targetEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget
{
	background-color: rgb(60,60,60);
	color: #fff;
	font: 9pt &quot;ç­‰çº¿l&quot;;
	border:0px solid rgb(0,0,0);
	border-radius:5px;
}
QWidget:focus
{
	background-color: rgb(60,60,60,0);
	color: #000;
	border:2px solid rgb(0,0,0);
	border-radius:5px;
}
</string>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="sourceEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget
{
	background-color: rgb(60,60,60);
	color: #fff;
	font: 9pt &quot;ç­‰çº¿l&quot;;
	border:0px solid rgb(0,0,0);
	border-radius:5px;
}
QWidget:focus
{
	background-color: rgb(60,60,60,0);
	color: #000;
	border:2px solid rgb(0,0,0);
	border-radius:5px;
}
</string>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2" colspan="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="confirmButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget
{
	background-color: rgb(255, 255, 255,0);
	border: 2px solid rgb(0,0,0);
	border-radius: 20px;
	color: rgb(0,0,0);
	font: 75 11pt &quot;ç­‰çº¿&quot;;
}
QWidget:hover
{
	font-size:12pt;
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}
QWidget:pressed 
{
	font-size:9pt;
}</string>
         </property>
         <property name="text">
          <string>confirm</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="fileListWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget
{
	background-color: rgb(60,60,60);
	border: 0px solid rgb(255,255,255);
	border-top-left-radius: 7px;
	border-bottom-left-radius: 7px;
	border-right-width:0px;
	color: #fff;
	font: 75 8pt &quot;å¾®è½¯é›…é»‘&quot;;
}
QListWidget:hover
{
	background-color:rgb(100,100,100);
}
QListWidget::item:hover
{
	background:rgb(40,40,40);
}
QListWidget::item:selected 
{
	border:2px solid #fff;
	border-radius:7px;
}
QListWidget::item
{
	color: #ff8200;
}</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="tagListWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget
{
	background-color: rgb(60,60,60);
	border: 0px solid rgb(0,0,0);
	border-top-right-radius: 7px;
	border-bottom-right-radius: 7px;
	border-left-width:0px;
	color: #fff;
	font: 75 8pt &quot;å¾®è½¯é›…é»‘&quot;;
}
QWidget:hover
{
	background-color:rgb(100,100,100);
}
QListWidget::item:hover
{
	background:rgb(40,40,40);
}
QListWidget::item:selected 
{
	border:2px solid #fff;
	border-radius:7px;
}
QListWidget::item
{
	color:#00df43;
}</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
